pkg/discovery/dtofactory/general_builder.go

```diff
@@ -21,10 +21,10 @@ var (
 		metrics.CPUProvisioned:     proto.CommodityDTO_CPU_PROVISIONED,
 		metrics.MemoryProvisioned:  proto.CommodityDTO_MEM_PROVISIONED,
 		metrics.Transaction:        proto.CommodityDTO_TRANSACTION,
-		metrics.CPUQuota:           proto.CommodityDTO_CPU_ALLOCATION,
-		metrics.MemoryQuota:        proto.CommodityDTO_MEM_ALLOCATION,
-		metrics.CPURequestQuota:    proto.CommodityDTO_CPU_REQUEST_ALLOCATION,
-		metrics.MemoryRequestQuota: proto.CommodityDTO_MEM_REQUEST_ALLOCATION,
+		metrics.CPULimitQuota:      proto.CommodityDTO_VCPU_LIMIT_QUOTA,
+		metrics.MemoryLimitQuota:   proto.CommodityDTO_VMEM_REQUEST_QUOTA,
```

Comment:
There's a typo here. Should be `CommodityDTO_VMEM_LIMIT_QUOTA` instead of `CommodityDTO_VMEM_REQUEST_QUOTA`. Will fix this in the next commit and will also update vendor with latest turbo-go-sdk changes to fix the failed CI build.